src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -1147,6 +1147,6 @@
   "block.structurize.framed_cactus_cobble_stone_timber_frame": "Framed Cactus Cobblestone Timber Frame",
   "block.structurize.multiblock": "Multi-Piston",
   "block.structurize.horizontal_plain_cactus_birch_timber_frame": "Plain Horizontal Cactus Birch Timber Frame",
-  "com.structurize.gui.buildtool.tip": "ESC to leave GUI to inspect preview",
-  "com.structurize.gui.buildtool.leave.tip": "Right click Building Tool on solid block to adjust position"
-}
\ No newline at end of file
+  "com.structurize.gui.buildtool.tip": "ESC to leave GUI to inspect the preview.",
+  "com.structurize.gui.buildtool.leave.tip": "Right-click the building tool to adjust the position of the schematic."
```

Comment:
Build Tool should be capitalized like any other item

Reply:
Other items are capitalized?

Reply:
You made something capitalized :D so be consistent I guess :D

Reply:
I made the item name capitalized because I figured it was like a title. Is there another place where I capitalized something?

Reply:
When referring something it should be typed the same way as the original one, imho

Reply:
Even if the original one is a title (I think)?

Reply:
Imho it's not a title, it's a name